platform/o.n.bootstrap/src/org/netbeans/NbClipboard.java

```diff
@@ -419,12 +422,33 @@ private final class GetContents implements Runnable {
         public GetContents(boolean notify) {
             this.notify = notify;
         }
-        
+
         @Override
+        @SuppressWarnings("SleepWhileInLoop")
         public void run() {
-            log.fine("Running update");
+            log.fine("Running update"); // NOI18N
             try {
-                Transferable transferable = systemClipboard.getContents(this);
+                Transferable transferable = null;
+                // There can be a race between multiple applications accessing
+                // the clipboard. If access can't be optained directly, retry
+                // for a maximum of 1s. This is called from the requestprocessor
+                // that is used because accessing the clipboard can block
+                // indefinitely. Running the access loop here is deemed similar
+                // in nature.
+                final int MAX_TRIES = 50;
```

Comment:
Is there a reason to handle this issue differently in `GetContents` from `SetContents`?  I think without good reason they should handle this in the same way, whether it be this way or that way being open to question.

https://github.com/apache/netbeans/blob/master/platform/o.n.bootstrap/src/org/netbeans/NbClipboard.java#L458

The pro and con of rescheduling rather than looping is that anything waiting on the task won't wait for the rescheduled one.